kroxylicious-docs/docs/_modules/authorization/con-authorization-rules.adoc

```diff
@@ -0,0 +1,95 @@
+:_mod-docs-content-type: CONCEPT
+
+
+[id='con-authorization-rules-{context}']
+= Authorization Rules
+
+The authorization rules define who is allowed to do what.
```

Comment:
I don't know how to convey this neatly, but it feels a bit clunky launching into rule/import syntax without some higher level description saying something about how we use a custom language for defining ACL rule and the user will write their rules in this form.

Maybe we could instead lead this section with an example and break down the bits of syntax? Having the ACL language examples away from `Authorization Rules` feels a little disconnected.

Reply:
it makes sense to have the configuration example, but maybe we should additionally have ACL rules examples